### DISCUSSION TITLE
Can you disable the volumes and segmentation server access?

### DISCUSSION THREAD

**Discussion by roccomoretti** [molstar/molstar#870] [General]
When running the viewer, I'm getting errors related to the volume and segmentation server integration. (In particular, on my machine, for the way I'm running things, the addresses being fetched aren't working.) This isn't something that I'm using at the moment, so I'm hoping I can just turn it off.

I've figured out that I can change which server it accesses by using the `volumesAndSegmentationsDefaultServer` parameter of `Viewer.create()`, but I can't seem to find the setting which deactivates attempting to access things altogether. Is there one?

**Comment by AustereTriceratops** [comment]
<img width="365" alt="Screen Shot 2023-07-12 at 11 07 16 AM" src="https://github.com/molstar/molstar/assets/39139710/bbe36027-0c4b-46cf-86fc-03387092117c">

Is this the error you're seeing? I've been running into this as well.

**Comment by dsehnal** [comment]
Good point. I've added the ability to disable extensions for the default viewer in https://github.com/molstar/molstar/pull/872. Should go in with the new release.
